spec/index.bs

```diff
@@ -7292,47 +7278,86 @@ dictionary GPURenderPassDepthStencilAttachment {
          a single [=mipmap level=], single [=array layer=], and all the [=aspects=] in the texture.
     - |this|.{{GPURenderPassDepthStencilAttachment/view}}.{{GPUTexture/[[descriptor]]}}.{{GPUTextureDescriptor/usage}}
         must contain {{GPUTextureUsage/RENDER_ATTACHMENT}}.
-    - If |this|.{{GPURenderPassDepthStencilAttachment/view}}.{{GPUTextureView/[[descriptor]]}}.{{GPUTextureViewDescriptor/format}}
-        contains both depth and stencil aspects (See [[#depth-formats|depth-stencil formats]]):
-        - |this|.{{GPURenderPassDepthStencilAttachment/depthReadOnly}} must be equal to |this|.{{GPURenderPassDepthStencilAttachment/stencilReadOnly}}
-    - If |this|.{{GPURenderPassDepthStencilAttachment/depthReadOnly}} is `true`:
-        - |this|.{{GPURenderPassDepthStencilAttachment/depthLoadValue}} must be {{GPULoadOp/"load"}}.
-        - |this|.{{GPURenderPassDepthStencilAttachment/depthStoreOp}} must be {{GPUStoreOp/"store"}}.
-    - If |this|.{{GPURenderPassDepthStencilAttachment/stencilReadOnly}} is `true`:
-        - |this|.{{GPURenderPassDepthStencilAttachment/stencilLoadValue}} must be {{GPULoadOp/"load"}}.
-        - |this|.{{GPURenderPassDepthStencilAttachment/stencilStoreOp}} must be {{GPUStoreOp/"store"}}.
+    - Let |format| be |this|.{{GPURenderPassDepthStencilAttachment/view}}.{{GPUTextureView/[[descriptor]]}}.{{GPUTextureViewDescriptor/format}}.
+    - If |format| contains a depth aspect:
+        |this|.{{GPURenderPassDepthStencilAttachment/depthMode}} must not be undefined.
+    - If |format| contains a stencil aspect:
+        |this|.{{GPURenderPassDepthStencilAttachment/stencilMode}} must not be undefined.
+    - If |format| contains both depth and stencil aspects (See [[#depth-formats|depth-stencil formats]]):
+        |this|.{{GPURenderPassDepthStencilAttachment/depthMode}} and
+        |this|.{{GPURenderPassDepthStencilAttachment/stencilMode}} must either:
+        - both be {{GPURenderPassDepthStencilAttachmentMode/"read-only"}}, or
+        - neither be {{GPURenderPassDepthStencilAttachmentMode/"read-only"}}.
 
     Issue: Describe the remaining validation rules for this type.
 </div>
 
 #### Load &amp; Store Operations #### {#load-and-store-ops}
 
+Note:
+On some GPU hardware (primarily mobile), providing a clear-value is significantly cheaper
+because it avoids loading data from main memory into tile-local memory.
+On other GPU hardware, there isn't a significant difference. As a result, it is
+recommended to use a "clear" mode, rather than a "load" mode, in cases where the
```

Comment:
It seems nit-picky, but this change uses terms like "a 'clear' mode" in several places, but never states what modes those are. I know it should be pretty obvious from context, but I think it would be beneficial to list out which modes are "clear" modes in at least one location, probably here.